### DISCUSSION TITLE
Possible to remove all style color on paste

### DISCUSSION THREAD

**Discussion by MrNuTruT** [sibiraj-s/ngx-editor#446] [General]
Hello,

it's possible to remove all style color and background color on paste on editor ?

Thx

**Comment by sibiraj-s** [comment]
Hi,

For that you will need to write a prosemirror plugin.

Some references to start.

- [transformPastedText](https://prosemirror.net/docs/ref/#view.EditorProps.transformPastedText)
- [transformPastedHTML](https://prosemirror.net/docs/ref/#view.EditorProps.transformPastedHTML)
- [transformPasted](https://prosemirror.net/docs/ref/#view.EditorProps.transformPasted)
